doc/whats_new.rst

```diff
@@ -539,6 +539,11 @@ API changes summary
      - ``utils.stats.rankdata``
      - ``neighbors.approximate.LSHForest``
 
+    - Deprecate the ``y`` parameter in :meth:`transform`.
```

Comment:
and :meth:`inverse_transform`

not sure this renders ok though as it should not point to anything specific